src/current/v23.1/monitoring-and-alerting.md

```diff
@@ -189,6 +189,685 @@ If you rely on external tools for storing and visualizing your cluster's time-se
+| `config.gcPolicy`              | `{...}`                                                      | An object representing the garbage collection settings for the range (e.g. [`gc.ttlseconds`](configure-replication-zones.html#gc-ttlseconds)). |
+| `config.globalReads`           | `false`                                                      | Whether the range enables fast [global reads](configure-replication-zones.html#global_reads).                                                  |
+| `config.numReplicas`           | `9`                                                          | The [replication factor](configure-replication-zones.html#num_replicas) for the range.                                                         |
+| `config.numVoters`             | `0`                                                          | The [number of voting replicas](configure-replication-zones.html#num_voters) for the range.                                                    |
+| `config.constraints`           | `[...]`                                                      | The [constraints](configure-replication-zones.html#constraints) for the range.                                                                 |
+| `config.voterConstraints`      | `[...]`                                                      | The [voting replica constraints](configure-replication-zones.html#voter_constraints) for the range.                                            |
+| `config.leasePreferences`      | `[...]`                                                      | The [lease preferences](configure-replication-zones.html#lease_preferences) for the range.                                                     |
+| `config.rangefeedEnabled`      | `false`                                                      | Whether [rangefeeds](create-and-configure-changefeeds.html#enable-rangefeeds) are enabled for this range.                                      |
+| `config.excludeDataFromBackup` | `false`                                                      | Whether this range's data should be excluded from [backups](backup.html).                                                                      |
+
+#### Examples
+
+- [Replication status - normal](#replication-status-normal)
+- [Replication status - constraint violation](#replication-status-constraint-violation)
+- [Replication status - under-replicated ranges](#replication-status-under-replicated-ranges)
+- [Replication status - ranges in critical localities](#replication-status-ranges-in-critical-localities)
+
+##### Replication status - normal
+
+The following example assumes you are running a newly started, local [`cockroach demo`](cockroach-demo.html) cluster started using the following command:
```

Comment:
In case you want to avoid the repetition of "newly started ... started using" maybe "newly started ... created using"? I don't feel super strongly about this.

```suggestion
The following example assumes you are running a newly started, local [`cockroach demo`](cockroach-demo.html) cluster started using the following command:
```